applications/app/views/signup/newsletterContent.scala.html

```diff
@@ -13,7 +13,7 @@
     </div>
     <div class="newsletter-card__wrapper">
         @emailListings.zipWithRowInfo.map { case (emailListing, row) =>
-            <div class="newsletter-card" data-component="newsletter-card @emailListing.name">
+            <div class="newsletter-card" data-component="newsletter-card @emailListing.identityNameOverride.getOrElse(emailListing.name)">
```

Comment:
I know I said the opposite earlier, but I would stick with `identityName` - it means we start again with a bit more of a clean, consistent slate.

Reply:
Works for me. I've reverted the change and gone back to using `identityName`